webdriver/tests/bidi/browsing_context/locate_nodes/invalid.py

```diff
@@ -218,3 +218,24 @@ async def test_params_start_nodes_dom_node_not_element(
             locator={"type": "css", "value": "div"},
             start_nodes=[remote_reference],
         )
+
+
+@pytest.mark.parametrize("domain", ["", "alt"], ids=["same_origin", "cross_origin"])
```

Comment:
For this invalid case I don't see the need to check cross origin. Is there a reason why you added it?

Reply:
(still lgtm though)

Reply:
I think we should check the happy case for both same-process and OOPiF, while keep invalid arg test generic. I can hardly imagine significant regression here.